website/docs/python.md

```diff
@@ -5,22 +5,40 @@ description: Python/pip dependencies support in Renovate
 
 # Python Package Manager Support
 
-Renovate supports upgrading dependencies in `pip` requirements (e.g. `requirements.txt`, `requirements.pip`) files.
+Renovate supports the following Python package managers:
+
+- `pip` (e.g. `requirements.txt`, `requirements.pip`) files
+- `pipenv` (e.g. `Pipfile`)
+- `setup.py`
```

Comment:
@rarkins Really? Cool!

Reply:
I mean re. `setup.py` in particular.

Reply:
If you can test it “in production” and let me know if it works for you then that would great. Once I get some feedback then I’ll default it to enabled